src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -59,15 +71,28 @@ private void configureGameBehavior() {
    * Configures subsystem default commands. Default commands are scheduled when no other command is
    * running on a subsystem.
    */
-  private void configureSubsystemDefaults() {}
+  private void configureSubsystemDefaults() {
+    hopper.setDefaultCommand(hopper.stop());
```

Comment:
i think i said this somewhere else, so sorry for the redundancy, but you can put these in the constructors for intake and hopper